packages/idos-store/index.ts

```diff
@@ -5,10 +5,11 @@ interface PipeCodecArgs<T> {
 
 export class Store {
   keyPrefix = "idOS-";
-
+  handler: Storage;
```

Comment:
Naming nit: I have (metaphorical) PTSD with the `handler` name. 😅 

Maybe we can call it `underlying_device` or something? It's also not super obvious to me. Happy to bike-shed in a Slack thread.

Reply:
We settled on `device`.